### ISSUE TITLE
Keyboard accessibility is broken

### ISSUE THREAD

**Issue by abitrolly** [canonical-websites/tutorials.ubuntu.com#615]
Keyboard navigation doesn't work in tutorial. Using Firefox 57 it is impossible to focus with Tab on content area to scroll it with up and down arrows. Only clicking with mouse there helps.

**Comment by steverydz** [comment]
Pull request: https://github.com/canonical-websites/tutorials.ubuntu.com/pull/680
